Code/training_common_utils.py

```diff
@@ -4,15 +4,26 @@
 import pandas as pd
 import numpy as np
 import tensorflow as tf
-from CheXpertDataset import ChexpertDataset, UZerosTransform, UOnesTransform, ReplaceNaNTransform, GreyScale_to_RGB
+from CheXpertDataset import ChexpertDataset, \
+    UZerosTransform, UOnesTransform, ReplaceNaNTransform
+from ImageTransforms import PILToNumpy, FixRatioResize, EqualizeHistogram, Gray2RGB, GaussianBlur
 from torchvision import transforms
 from torch.utils.data import Subset, DataLoader
 
 image_preprocessing = transforms.Compose([
-    # GreyScale_to_RGB(),
-    transforms.Resize((600, 600)),
+    PILToNumpy(),
+    EqualizeHistogram(),
+    Gray2RGB(),
+    FixRatioResize(512, 128),
```

Comment:
Does this need to be a tuple, like `((512, 128))`?  Also, it seems like line 61 expects a 3-tuple.  Maybe these aren't actually issues in Python, but wanted to bring up.

Reply:
No? I wrote that one. It's in the ImageTransforms file now.

Reply:
Not sure which line 61 your referring too?
the normalize on line 20 is tuples but thats provided by the torchvision transforms, and isn't something we wrote.

Reply:
This line 61
https://github.com/ml2-final-project/Final-Project-Group8/pull/9/files#diff-37dd26a9587a0207f2418ef8c4462628R61

Also, I was concerned that this only takes one arg (aside from self)
https://github.com/ml2-final-project/Final-Project-Group8/pull/9/files#diff-37dd26a9587a0207f2418ef8c4462628R45

Reply:
got it. remember that when we construct a composed transform we are constructing the objects (`__init__` function gets called), not actually invoking the transformation code (`__call__`). 
As such, I'm effectively using init to configure a thing that does the transformation, and only passing args to it when it needs some configuration regarding the behavior of the transform.
The only argument line 45 needs to take in is the image, as all our image transforms do. the `init` function on line 41 of the same file is the function called when we build the composed transform.